FWCore/Framework/src/stream/ProducingModuleAdaptorBase.cc

```diff
@@ -224,13 +226,13 @@ namespace edm {
 
     template <typename T>
     void ProducingModuleAdaptorBase<T>::doStreamEndLuminosityBlock(StreamID id,
-                                                                   LuminosityBlockPrincipal const& lbp,
-                                                                   EventSetupImpl const& ci,
+                                                                   LumiTransitionInfo const& info,
                                                                    ModuleCallingContext const* mcc) {
+      LuminosityBlockPrincipal const& lbp = info.principal();
```

Comment:
I suppose this line could be omitted and two lines below the `LuminosityBlock`  constructed as
```cpp
      LuminosityBlock lb(info, moduleDescription_, mcc, true);
```

Reply:
Done. Thanks.